pkg/controllers/workload_reconciler.go

```diff
@@ -212,6 +214,33 @@ func (r *WorkloadReconciler) isSupplyChainReady(supplyChain *v1alpha1.ClusterSup
 	return supplyChainReadyCondition.Status == "True"
 }
 
+func buildWorkloadResourceLabeler(owner, blueprint client.Object) realizer.ResourceLabeler {
```

Comment:
nit: "labe**ll**er" instead of "labeler" (I think)

Reply:
This is goland's dictionary which is American: https://grammarist.com/spelling/label/#:~:text=In%20American%20English%2C%20label%20makes,extends%20to%20labelers%20and%20labellers.